blog/_posts/2020-03-31-extensions.md

```diff
@@ -0,0 +1,52 @@
+---
+slug: more-customizable-tiltfiles-with-extensions
+date: 2020-04-01
+author: victor
+layout: blog
+title: "More customizable Tiltfiles with Extensions"
+subtitle: "Share Tiltfile code snippets with the Tilt community in a streamlined platform. And get a free t-shirt!"
+image: "/assets/images/more-customizable-tiltfiles-with-extensions/michael-dziedzic-XTblNijO9IE-unsplash.jpg"
+image_caption: "Photo by <a href='https://unsplash.com/@lazycreekimages?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText'>Michael Dziedzic</a> on <a href='https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText'>Unsplash</a>"
+tags:
+  - tilt
+keywords:
+  - extensions
+  - tilt
+  - tiltfile
+---
+
+Tilt lets you declare your dev environment in Starlark, a subset of Python. But Starlark is missing a big feature! There's no package manager. Starlark doesn't have built-in ways to share your configuration.
```

Comment:
Let's open with an active voice statement.

```suggestion
In Tilt, you declare your dev environment in Starlark, a subset of Python. But Starlark is missing a big feature: there's no package manager! Starlark doesn't have built-in ways to share your configuration.
```

One more thing, I find that the word `configuration` sounds really weird to me. It doesn't tell me what it is that I'd like to share. Am I sharing a file? A function? What is the tangible item being shared?

Reply:
I updated to be "Starlark doesn't have built-in ways to share code."